frontend/src/component/feature/FeatureView/FeatureView.tsx

```diff
@@ -185,6 +187,15 @@ export const FeatureView = () => {
         return <div ref={ref} />;
     }
 
+    const handleCopyToClipboard = () => {
+        copy(feature.name);
+        setIsFeatureNameCopied(true);
+
+        setTimeout(() => {
+            setIsFeatureNameCopied(false);
+        }, 3000);
+    };
```

Comment:
I'd like this to be wrapped in a try/catch and handle the case where copying fails.

Reply:
That's a valid callout. I will cook it up.

Reply:
I have wrapped the copy under try-catch